guidelines/MAINTAINERS-guidelines.md

```diff
@@ -6,60 +6,97 @@ grand_parent: Hyperledger TOC
+The following provides guidelines and suggested content to include in the `MAINTAINERS` file.
+
+## Maintainer Scopes and GitHub Roles
+
+Becoming a Maintainer for a repository implies being granted special privileges within that repository to do the additional tasks required of a Maintainer. In most cases, the additional privileges are implemented
+by giving maintainers one of the elevated [GitHub roles](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization) within the repository.
+
+In most repositories, only the "Maintain" GitHub role is used. It is given to all repository Maintainers.
+
+In some repositories, the maintainers may define a list of "scopes" and each Maintainer given one or more of the designated scopes:
+
+- In rare cases, Maintainers in a repository may request that the Hyperledger Foundation GitHub organization administrators enable designating some Maintainers with different GitHub roles, such as "Admin" (more capable than "Maintain") or "Triage" (less capable than "Maintain").
+  - Each elevated GitHub role needed in a repository requires the Hyperledger Foundation GitHub organization administrators to create and maintain an additional GitHub Team.
+  - The Hyperledger Foundation GitHub organization administrators manage a team per Hyperledger Project that includes all contributors to the project. That team is given the "Read" GitHub role in all project repositories. This team need not be documented in the "MAINTAINERS" file.
+- Maintainers **MAY** define Maintainers scope within a repository don't require
+  elevated GitHub privileges. For example, a scope might include hosting
+  community meetings, or contributing to the Hyperledger Project's Quarterly
+  report.
+
+If there is more than the single "Maintainer" scope used in a repository, there **MUST** be a list of the repository specific scopes in the MAINTAINERS file. The list must include the scope name, the definition of the scope, and if applicable, the related GitHub Role and Team for the scope.
+
+| Scope | Definition | GitHub Role | GitHub Team |
+| ----- | ---------- | ----------- | ----------- |
+|       |            |             |             |
+
+## List of Repository Maintainers
+
+Lists of active and emeritus maintainers MUST be included in the `MAINTAINERS` file.
+
+Changes to the maintainers lists MUST be made via Pull Requests. Once a new `MAINTAINERS` file is created or a PR changing the maintainer lists within the file is merged, a corresponding update to the affected GitHub Teams within the Hyperledger GitHub organization must be made. This is a manual process and the maintainers must ensure that it occurs.
+
+It is recommended that the lists be sorted alphabetically and contain at least the maintainers name and GitHub ID. If useful, other columns may be added to the table, such as LFID, Discord ID, Email, and Company Affiliation.
```

Comment:
This is a change from what was previously required. There were reasons for the other fields that I believe are still relevant. Please see my other comment for details.